data.sql

```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutured, escape_attempts)
+VALUES
+('Agumon', '2020-02-03', 10.23, true, 0),
+('Gabumon', '2018-11-15', 8, true, 2),
+('Pikachu', '2021-01-07', 15.04, false, 1),
+('Devimon', '2017-05-07', 11, true, 5);
```

Comment:
| Requirements 📝 |
| --- |
|![image](https://github.com/kwesiObuobi/vet-clinic/assets/93039359/1ba4687d-b9cf-462a-a045-f5fc3ab06854) |

- [ ] It would be nice if you follow the requirements because right now the date_of_birth of Devimon is not correct 👍 It should be `2017-05-12`

Reply:
Oops, thanks for flagging this.
My mistake 😅